playbooks/roles/install-wetty/tasks/wetty.yml

```diff
@@ -0,0 +1,266 @@
+      -newkey rsa:2048
+      -keyout /etc/letsencrypt/live/{{ openshift_public_hostname }}/privkey.pem
+      -out /etc/letsencrypt/live/{{ openshift_public_hostname }}/cert.pem
+  args:
+    creates: /etc/letsencrypt/live/{{ openshift_public_hostname }}/*.pem
+  become: true
+  when: not letsencrypt_cert_generation and not cert_directory.stat.exists
+
+- name: Give the {{ ansible_user }} access to the certs
+  acl:
+    path: "/etc/letsencrypt"
+    entity: "{{ ansible_user }}"
+    etype: user
+    permissions: rx
+    recursive: yes
+    follow: yes
+    state: present
+  become: true
+
+- name: Install wetty globaly
+  command: npm install wetty -g chdir=/home/{{ ansible_user }}
+  args:
+    creates: /usr/lib/node_modules/wetty/bin/wetty.js
+  become: true
+
+- name: Install the patch package
+  package:
+    name: patch
+    state: latest
+  become: true
+
+- name: Patch hterm_all.js to handle iOS spaces
+  patch:
+    backup: yes
+    basedir: /
+    src: hterm_all.js-iOS-patch
+  become: true
+
+- name: Create temporary certificate file
+  tempfile:
+    state: file
+    suffix: cert
+  register: wetty_cert
+
+- name: Append wetty certificate(s)
+  shell: "cat {{ item }} >> {{ wetty_cert.path }}"
+  with_items:
+    - "/etc/letsencrypt/live/{{ openshift_public_hostname }}/cert.pem"
+
+- name: Set certificate permissions
+  file:
+    path: "{{ wetty_cert.path }}"
+    group: "{{ ansible_user }}"
+    owner: "{{ ansible_user }}"
+    mode: 0644
+  become: true
+
+- name: Copy certificate file
+  copy:
+    src: "{{ wetty_cert.path }}"
+    dest: "/home/{{ ansible_user }}/.ssh/cert.pem"
+    remote_src: yes
+
+- name: Create temporary key file
+  tempfile:
+    state: file
+    suffix: cert
+  register: wetty_privkey
+
+- name: Append wetty key(s)
+  shell: "cat {{ item }} >> {{ wetty_privkey.path }}"
+  with_items:
+    - "/etc/letsencrypt/live/{{ openshift_public_hostname }}/privkey.pem"
+
+- name: Set key permissions
+  file:
+    path: "{{ wetty_privkey.path }}"
+    group: "{{ ansible_user }}"
+    owner: "{{ ansible_user }}"
+    mode: 0644
+  become: true
+
+- name: Copy key file
+  copy:
+    src: "{{ wetty_privkey.path }}"
+    dest: "/home/{{ ansible_user }}/.ssh/privkey.pem"
+    remote_src: yes
+
+- name: Set certificate permissions
+  file:
+    path: "/home/{{ ansible_user }}/.ssh/{{ item }}"
+    group: "{{ ansible_user }}"
+    owner: "{{ ansible_user }}"
+    mode: 0400
+  become: true
+  with_items:
+    - cert.pem
+    - privkey.pem
+
+- name: Template systemd service file
+  template:
+    src:   "wetty.service.j2"
+    dest:  "/etc/systemd/system/wetty.service"
+    group: root
+    owner: root
+    mode:  0644
+  become: true
+
+- name: Reload systemd units
+  command: systemctl daemon-reload
+  register: rd_result
+  become: true
+
+- name: Make sure a service is running
```

Comment:
systemd daemon-reload will register unit file changes w/ init and reload cgroup hierarchy, but will not enforce unit file changes without forcing service restart. We need to restart wetty.service not just make sure it's running here.